modules/ROOT/pages/policies-policy-categories.adoc

```diff
@@ -0,0 +1,40 @@
+= Policy Categories
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies are categorized by the function they perform. For example, because you are able to resolve issues with the help of logs, the Message Logging policy is categorized as a troubleshooting policy. The following table lists the Default policy name, its category and the function that it performs. +
+You can apply only one policy with a particular `providedCharacteristics` at a given time in an API.
+
+//Fede--what would 'providedCharacteristics' translate to for the customer?. I mean they need to understand what I'm talking about here right?
+
+[%header,cols="40a,15a,60a"]
+|===
+| Policy | Category | Function
+| Basic Authentication - LDAP | Security | Authenticates the LDAP credentials.
+| Basic Authentication - Simple | Security | Authenticates single user password.
+| Client ID Enforcement | Compliance | Allows access to client applications with a valid client credentials.
```

Comment:
```suggestion
| Client ID Enforcement | Compliance | Allows access to client applications with valid client credentials.
```

Reply:
It was correct in my version. Done! Thanks!